### ISSUE TITLE
Ease customizing which bazel versions one tests against by default

### ISSUE THREAD

**Issue by ittaiz** [bazelbuild/bazel-integration-testing#48]
currently you can be very explicit and control the bazel version or just use the default versions from the library which is a lot.
I'd like it to be easy to define your versions and have that be used by default.
Not sure how.
@damienmg wdyt?

**Comment by damienmg** [comment]
You mean like I want to test every thing starting bazel 0.7 for instance? That would make sense

Maybe we should also make sure we do not ship several minor of the same version of Bazel. E.g we should provide only 0.10.1 not 0.10.0

**Comment by ittaiz** [comment]
> You mean like I want to test every thing starting bazel 0.7 for instance? That would make sense

Yes, that's one option. Another is to say that my entire repository only checks with 0.11.1 because that's what my organization uses (relevant for people who write internal rules).

> Maybe we should also make sure we do not ship several minor of the same version of Bazel. E.g we should provide only 0.10.1 not 0.10.0

Sounds good. I'll wait for @maximn to finish with his changes and I'll update the default versions
